plugin.video.twitch/addon.xml

```diff
@@ -1,27 +1,30 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
-<addon id='plugin.video.twitch' version='1.3.1' name='TwitchTV' provider-name='StateOfTheArt and ccaspers'>
-  <requires>
-    <import addon='xbmc.python' version='2.1.0'/>
-    <import addon='script.module.simplejson' version='2.0.10'/>
-    <import addon="script.module.xbmcswift2" version="1.2.0"/>
-  </requires>
-  <extension point='xbmc.python.pluginsource' library='default.py'>
+<addon id='plugin.video.twitch' version='1.4.0' name='Twitch' provider-name='StateOfTheArt and ccaspers'>
+    <requires>
+        <import addon='xbmc.python' version='2.20.0'/>
```

Comment:
You are asking for this PR to be pulled into the "Gotham" repo, but you have the 'xbmc.python' version set to Isengard. Which version repo is this update intended for?